ocs_ci/ocs/ui/base_ui.py

```diff
@@ -255,10 +259,24 @@ def take_screenshot(self):
             self.screenshots_folder,
             f"{datetime.datetime.now().strftime('%Y-%m-%dT%H-%M-%S.%f')}.png",
         )
-        logger.info(f"Creating snapshot: {filename}")
+        logger.info(f"Creating screenshot: {filename}")
         self.driver.save_screenshot(filename)
         time.sleep(0.5)
 
+    def copy_dom(self):
+        """
+        Get page source of the webpage
+
+        """
+        filename = os.path.join(
+            self.screenshots_folder,
```

Comment:
Dom is not screenshot, please rename it to something mor generic like ui_logs_dir make some structure like we talked in DM

Reply:
done